prac_05/hex_colours.py

```diff
@@ -0,0 +1,29 @@
+COLOURS_CODES = {
```

Comment:
Using the guidelines for the subject dictionaries are named as key_to_value therefore to meet the guideline COLOURS_CODES should be something like COLOURS_TO_CODES